### PR TITLE
widelands: update to 1.0.

### PR DESCRIPTION
**Pull request by travankor**
#### General
- [ ] This is a new package and it conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements)

#### Have the results of the proposed changes been tested?
- [ ] I use the packages affected by the proposed changes on a regular basis and confirm this PR works for me
- [X] I generally don't use the affected packages but briefly tested this PR

### CODE REVIEW

File: srcpkgs/widelands/template [hunk 1]
```diff
@@ -1,21 +1,23 @@
 # Template file for 'widelands'
 pkgname=widelands
-version=21
-revision=5
-wrksrc="${pkgname}-build${version}"
+# Widelands 1.0 release supercedes beta builds
+reverts="21_5"
```
Comment (paper42) [inline srcpkgs/widelands/template:4]: this is missing many versions (and revisions) that were packaged

File: srcpkgs/widelands/template [hunk 2]
```diff
@@ -1,21 +1,23 @@
...
+version=1.0
+revision=1
+wrksrc="${pkgname}-${version}"
```
Comment (paper42) [inline srcpkgs/widelands/template:7]: this is the default value of $wrksrc, so you don't need to overwrite it

File: srcpkgs/widelands/template [hunk 3]
```diff
@@ -1,21 +1,23 @@
...
+ glu-devel glew-devel libcurl-devel SDL2-devel SDL2_gfx-devel SDL2_image-devel
  SDL2_net-devel SDL2_mixer-devel SDL2_ttf-devel"
 short_desc="Real-time strategy game"
-maintainer="Orphaned <orphan@voidlinux.org>"
+maintainer="travankor <travankor@tuta.io>"
 license="GPL-2.0-or-later"
 homepage="http://www.widelands.org/"
-distfiles="https://github.com/widelands/widelands/archive/build${version}.tar.gz>${pkgname}-${version}.tar.gz"
-checksum=601e0e4c6f91b3fb0ece2cd1b83ecfb02344a1b9194fbb70ef3f70e06994e357
+distfiles="https://github.com/widelands/widelands/archive/v${version}.tar.gz>${pkgname}-${version}.tar.gz"
```
Comment (paper42) [inline srcpkgs/widelands/template:19]: do we still need `>${pkgname}-${version}`?

File: srcpkgs/widelands/template (hunk 1 above)
Comment (travankor) [reply srcpkgs/widelands/template:4]: Should it revert every packaged version?

File: srcpkgs/widelands/template (hunk 3 above)
Comment (travankor) [reply srcpkgs/widelands/template:19]: `$wkrsrc` would be `v${version}` otherwise

File: srcpkgs/widelands/template (hunk 1 above)
Comment (travankor) [reply srcpkgs/widelands/template]: Think I got them all.

**Pull request merged**